cmd/rules/main.go

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/mainflux/mainflux"
+	authapi "github.com/mainflux/mainflux/auth/api/grpc"
+	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/rules"
+	"github.com/mainflux/mainflux/rules/api"
+	rehttpapi "github.com/mainflux/mainflux/rules/api/rules/http"
+	localusers "github.com/mainflux/mainflux/things/users"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
+	mfSDK "github.com/mainflux/mainflux/pkg/sdk/go"
+	opentracing "github.com/opentracing/opentracing-go"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+	jconfig "github.com/uber/jaeger-client-go/config"
+)
+
+const (
+	defLogLevel        = "info"
+	defHTTPPort        = "9099"
+	defKuiperURL       = "http://localhost:9081"
+	defJaegerURL       = ""
+	defServerCert      = ""
+	defServerKey       = ""
+	defSingleUserEmail = ""
+	defSingleUserToken = ""
+	defAuthURL         = "localhost:8181"
+	defAuthTimeout     = "1s"
+	defClientTLS       = "false"
+	defCACerts         = ""
+	defThingsLocation  = "http://localhost"
+
+	envLogLevel        = "MF_RE_LOG_LEVEL"
+	envHTTPPort        = "MF_RE_HTTP_PORT"
+	envKuiperURL       = "MF_KUIPER_URL"
+	envJaegerURL       = "MF_JAEGER_URL"
+	envServerCert      = "MF_RE_SERVER_CERT"
+	envServerKey       = "MF_RE_SERVER_KEY"
+	envSingleUserEmail = "MF_RE_SINGLE_USER_EMAIL"
+	envSingleUserToken = "MF_RE_SINGLE_USER_TOKEN"
+	envAuthURL         = "MF_AUTH_GRPC_URL"
+	envAuthTimeout     = "MF_AUTH_GRPC_TIMEOUT"
+	envClientTLS       = "MF_RE_CLIENT_TLS"
+	envCACerts         = "MF_RE_CA_CERTS"
+	envThingsLocation  = "MF_RE_THINGS_LOCATION"
```

Comment:
Rename this envars `MF_RULES_LOG_LEVEL` etc etc

Reply:
also, I would sort this envs for the sake of readability 
first all of the `MF_RULES...` than other.